modules/google-maps/src/google-maps-overlay.ts

```diff
@@ -215,7 +215,8 @@ export default class GoogleMapsOverlay {
     deck.setProps({
       width,
       height,
-      viewState: {altitude, repeat: true, ...rest}
+      // @ts-expect-error altitude is accepted by WebMercatorViewport but not exposed by type
+      viewState: {altitude, ...rest} as MapViewState
```

Comment:
Should this use `position: [0,0,altitude]` instead since that is on the type?

Reply:
No, this is a legacy parameter for calculating fov. I will address this properly in a separate PR.

Reply:
Sg. Does mapbox overlay need a type cast as well?